src/overlays/actors/ovl_Dm_Hina/z_dm_hina.h

```diff
@@ -10,10 +10,24 @@ typedef void (*DmHinaActionFunc)(struct DmHina*, PlayState*);
 typedef struct DmHina {
     /* 0x000 */ Actor actor;
     /* 0x144 */ DmHinaActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x10];
+    /* 0x148 */ f32 unk148;
+    /* 0x14C */ f32 unk14C;
+    /* 0x150 */ f32 unk150;
+    /* 0x154 */ f32 unk154;
     /* 0x158 */ f32 unk_158;
     /* 0x15C */ f32 unk_15C;
-    /* 0x160 */ char unk_160[0x24];
+    /* 0x160 */ f32 unk160;
+    /* 0x164 */ f32 unk164;
+    /* 0x168 */ f32 unk168;
+    /* 0x16C */ f32 unk16C;
+    /* 0x170 */ f32 unk170;
+    /* 0x174 */ f32 unk174;
+    /* 0x178 */ char pad178[2];
```

Comment:
```suggestion
    /* 0x178 */ UNK_TYPE1 pad178[2];
```
Small nit, but we like to use `UNK_TYPE1` for confirmed pads.